app/controllers/consent_forms/edit_controller.rb

```diff
@@ -4,10 +4,10 @@ class ConsentForms::EditController < ConsentForms::BaseController
 
   layout "two_thirds"
 
-  steps(*ConsentForm.form_steps)
-
   before_action :set_session
   before_action :set_consent_form
+  before_action :set_steps # Uses @consent_form, needs set_consent_form
```

Comment:
Could we exclude `update` from this action since we have to run `set_steps` manually there anyway? And would that obviate the need for `@wizard_translations = nil`?

```suggestion
  before_action :set_steps, except: :update # Uses @consent_form, needs set_consent_form
```

Reply:
Great idea, I'll try it and report back.